source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+| [**SSL Labs Rating**](https://www.ssllabs.com/ssltest/){.external}    | A                         | A+ [with HSTS](/docs/hsts/)     | A+ [with HSTS](/docs/hsts/) *         |
+| **Protocol**                                                          | TLS 1.1 & 1.2             | TLS 1.2 with SNI                | TLS 1.2 with SNI                      |
+| **Ciphers**                                                           | Weak 3DES Cipher          | No Weak 3DES cipher             | No Weak 3DES cipher                   |
+| **Delivery**                                                          | US Datacenter             | [Global CDN](/docs/global-cdn)  | [Global CDN](/docs/global-cdn)        |
+| **Encryption Endpoint**                                               | Load Balancer             | Application Container           | Application Container                 |
+
+\* Browser compatibility and SSL Labs score is guaranteed for shared Let’s Encrypt certificates. The same results are typical for a custom certificate from a mainstream CA with mainstream attributes, but not guaranteed.  For custom certificates, compatibility and SSL Labs score depends on attributes of that certificate, such as number of SAN entries, CA and signing algorithm.
+
+## Frequently Asked Questions
+
+### Do I need a separate certificate for each site in my organization?
+
+Nope! You can use the a single certificate to cover multiple domains spread across various environments or sites. This capability is enabled because the Global CDN uses a technology called Server Name Indication (SNI), which automatically matches inbound requests with an appropriate certificate, including custom certificates.
+
+### How long will it take to load my certificate into Pantheon?
+
+Please allow two business days to get a CSR and load the certificate.
+
+### What about for sites purchased online?
+
+Custom certificates are currently available to customers with a Pantheon contract using Legacy SSL. We have no current plans to offer it to Basic or Performance sites. If you pay for your site with a credit card and are interested in using a custom certificate, please [let us know](@ari-gold FEEEEED ME).
+
+### Will custom certificates be self-serve?
+
+The concierge service is designed to quickly guide you through the steps required to deliver HTTPS on the Global CDN using your custom certificate. Due to the limited demand, and infrequent need to load custom certificates, the concierge service may be the best fit, although we are evaluating following up with a self-serve option.
```

Comment:
Say no first, then go into the details second.

Reply:
How about: 
"We have no current plans to offer a self-serve option. The concierge service is designed to quickly guide you through the steps required to deliver HTTPS on the Global CDN using your custom certificate, and we may follow-up with a self-serve option in the future."